features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/TestConsole/TestConsole.jsx

```diff
@@ -419,175 +368,184 @@ class TestConsole extends React.Component {
     render() {
         const { classes } = this.props;
         const {
-            swagger, api, securitySchemeType, selectedEnvironment, productionAccessToken, sandboxAccessToken,
-            labels, environments, scopes, username, password, selectedKeyType, serverError, settings, host, baseUrl,
-            loading,
+            api, swagger, securitySchemeType, selectedEnvironment, environments, settings,
+            showToken, apiSettings,
         } = this.state;
-        if (serverError) {
-            return (
-                <Typography variant='h4' className={classes.titleSub}>
-                    {serverError}
-                </Typography>
-            );
-        }
-        if (!api || !swagger || !settings) {
+        const authorizationHeader = 'Internal-Key';
+        if (!api || !securitySchemeType || !selectedEnvironment || !environments || !swagger || !settings
+            || !apiSettings) {
             return <Progress />;
         }
-        let authorizationHeader = api.authorizationHeader ? api.authorizationHeader : 'Authorization';
-        authorizationHeader = 'testkey';
+        const authHeader = `${authorizationHeader}`;
```

Comment:
Do we have a switch or selector to select auth header like `Internal-key` , `Authorization` if not this is always `Internal-key` right ?

Reply:
yes, this is always Internal-Key.